index.js

```diff
@@ -464,7 +464,7 @@ module.exports = {
 
     let versionTree = writeFile(
       'ember-paper/register-version.js',
-      `Ember.libraries.register('Ember Paper', '${version}');`
+      `const Ember = require('ember').default;Ember.libraries.register('Ember Paper', '${version}');`
```

Comment:
I think I would prefer just to remove this functionality 🤔 I don't think we need to define the Ember-Paper version in the client side for any good reason